### PR TITLE
chore: Convert issue templates to issue forms, part 1

### PR DESCRIPTION
**Pull request by tumido**
Part of: https://github.com/operate-first/support/issues/292

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/onboarding_to_cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+              Please select a cluster from the list at
+              https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/moc#available-clusters
+          options:
+              - Zero
+              - Infra
+              - Rick
+    - type: input
+      id: team-name
+      attributes:
+          label:
```
Comment (HumairAK) [inline .github/ISSUE_TEMPLATE/onboarding_to_cluster.yaml:21]: looks like you're missing the label here? resulting in: 

image

Comment (tumido) [reply .github/ISSUE_TEMPLATE/onboarding_to_cluster.yaml]: resolves, thanks :+1:

File: .github/ISSUE_TEMPLATE/onboarding_to_cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    - type: input
+      id: project-name
+      attributes:
+          label: Desired project names
+          description: |
+              Resources in OpenShift are contained in "namespaces" (also called
+              "projects"). Unless you request otherwise, we will create a single
+              project for you with the same name as your selected team name.
+
+              If you need additional namespaces, or want a project name that
```
Comment (HumairAK) [inline .github/ISSUE_TEMPLATE/onboarding_to_cluster.yaml:48]: for later parsing, can we update this to say something like "if you need more than one namespace, please list them here, using a comma delimiter

**Pull request merged**